scripts/flattener.py

```diff
@@ -905,11 +885,11 @@ def main(mfinal_id):
 					download_file(mxr.get('s3_uri'), fm.MTX_DIR, mxr.get('accession'))
 
 			mxr_name = '{}.h5'.format(mxr_acc) if mxr['s3_uri'].endswith('h5') else '{}.h5ad'.format(mxr_acc)
-			if mfinal_obj.get('spatial_s3_uri', None) and mfinal_obj['assays'] == ['spatial transcriptomics']:
+			if GLOBAL_OBJS.MFINAL.mfinal_obj.get('spatial_s3_uri', None) and GLOBAL_OBJS.MFINAL.mfinal_obj['assays'] == ['spatial transcriptomics']:
```

Comment:
I ran the flattener for LATDF584NGT, and got the following error:

> Traceback (most recent call last):
  File "/Users/jychien/Lattice-Data/lattice-tools/scripts/flattener.py", line 1264, in <module>
    main(args.file)
  File "/Users/jychien/Lattice-Data/lattice-tools/scripts/flattener.py", line 898, in main
    if 'spatial' in mfinal_adata.uns.keys():
NameError: name 'mfinal_adata' is not defined

Reply:
Oh no I must've missed running LATDF584NGT..

For the caps question, I chose to capitalize since we are trying to use these as constants, but would definitely be open to changing it to only first letter caps.

It would be possible to have all the properties under global, I just thought we wanted more organization of the properties.. it does lead to less readability though. I can go back through and change it so that they're all under global if that's whats desired.